### PR TITLE
moved tickspersecond in damagezone.cs to public editor variable

### PR DESCRIPTION
**Pull request by rau98**

### CODE REVIEW

**Review by Meowkov — DISMISSED** [review]
Looking good.

File: src/Module.Server/Scripts/DamageZone.cs [hunk 1]
```diff
@@ -9,12 +9,13 @@ namespace Crpg.Module.Scripts;
 /// <summary>A script that damages agents inside the entity it is attached to.</summary>
 internal class DamageZone : ScriptComponentBehavior
 {
-    private const int TicksPerSecond = 2;
+    //private const int TicksPerSecond = 2;
```
Comment (Muparadzi) [inline src/Module.Server/Scripts/DamageZone.cs:12]: you can remove this line

**Review by Muparadzi — CHANGES_REQUESTED** [review]

**Review by Muparadzi — APPROVED** [review]
Thanks for your first PR!

**Review by Meowkov — APPROVED** [review]
Approved

**Review by or2e — APPROVED** [review]

**Review by Yeldur — APPROVED** [review]
approved

**Pull request merged**